projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -0,0 +1,391 @@
+<?php
+/**
+ * Help center
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+/**
+ * Class Help_Center
+ */
+class Help_Center {
+	/**
+	 * Class instance.
+	 *
+	 * @var Help_Center
+	 */
+	private static $instance = null;
+
+	/**
+	 * Asset file.
+	 *
+	 * @var asset_file
+	 */
+	private $asset_file;
+
+	/**
+	 * Version number of the plugin.
+	 *
+	 * @var version
+	 */
+	private $version;
+
+	/**
+	 * Help_Center constructor.
+	 */
+	public function __construct() {
+		global $wp_customize;
+
+		if ( isset( $wp_customize ) ) {
+			return;
+		}
+
+		if ( ! function_exists( 'wpcom_get_site_purchases' ) ) {
+			return;
+		}
+
+		// $request = wp_safe_remote_get( 'https://widgets.wp.com/help-center/help-center.asset.json' );
+		// if ( is_wp_error( $request ) ) {
+		// return;
+		// }
+		$this->asset_file = json_decode( '{"dependencies":["lodash","react","react-dom","wp-a11y","wp-api-fetch","wp-components","wp-compose","wp-data","wp-data-controls","wp-date","wp-element","wp-html-entities","wp-i18n","wp-keycodes","wp-polyfill","wp-primitives","wp-url"],"version":"f5cd531a3b8c6c8517ff"}', true );
```

Comment:
Should figure out a way to load the asset file. Loading it from widgets.wp.com works but it's annoying during development since it would be downloaded from production.